scripts/cnode-helper-scripts/env

```diff
@@ -1017,8 +1017,13 @@ case ${NWMAGIC} in
     NETWORK_NAME="Guild"
     SHELLEY_TRANS_EPOCH=2
     [[ -z ${KOIOS_API} ]] && KOIOS_API="https://guild.koios.rest/api/v0" ;;
+  2)
```

Comment:
Remove staging a few lines up?
Nobody runs this right?

Reply:
Good point, removed